src/EnergyPlus/HVACControllers.cc

```diff
@@ -1465,6 +1434,10 @@ namespace HVACControllers {
                 (0.001 / (2100.0 * max(ControllerProps(ControlNum).MaxVolFlowActuated, SmallWaterVolFlow))) * (HVACEnergyToler / 10.0);
             // do not let the controller tolerance exceed 1/10 of the loop temperature tolerance.
             ControllerProps(ControlNum).Offset = min(0.1 * HVACTemperatureToler, ControllerProps(ControlNum).Offset);
+            if (ControllerProps(ControlNum).ControlVar == HVACControllers::iTemperatureAndHumidityRatio) {
+                // for temperature and humidity control, need tighter tolerance if humidity control kicks in
+                ControllerProps(ControlNum).Offset = ControllerProps(ControlNum).Offset*0.1;
```

Comment:
Would be nice to be able to trigger a tighter tolerance only when HumRatCtrlOverride = true.

Reply:
I tried that first but it didn't work. There's some initial setup step that I couldn't find that includes the offset.